exercises/exercise01.py

```diff
@@ -0,0 +1,20 @@
+#solution on https://www.practicepython.org/exercise/2014/01/29/01-character-input.html 
+#and write the result to file result.txt
+
+from datetime import date
+
+
+print('Enter your name: ')
+name = input()
+print('Enter your age: ')
+age = int(input())
+
+today = date.today()
+currentYear = today.strftime("%Y")
+result = name + ' will turn 100 years old in ' + str(int(currentYear )+ 100 - age)
```

Comment:
can write like result = f"{name} will turn 100 years old in {currentYear + 100 - age}"